pkg/cluster/cluster.go

```diff
@@ -760,6 +763,17 @@ func (c *Cluster) initHumanUsers() error {
 	return nil
 }
 
+func (c *Cluster) initHumanUsers() error {
+
+	for _, postgresAdminTeam := range c.OpConfig.PostgresAdminTeams {
+		c.initTeamMembers(postgresAdminTeam, true /*isPostgresAdminTeam*/)
```

Comment:
Error return value of `c.initTeamMembers` is not checked

Reply:
Imho no need for this comment in the function call.